### DISCUSSION TITLE
Need help to Learn & Understand the project.

### DISCUSSION THREAD

**Discussion by Chandra-Sekhar-Bala** [horizontalsystems/unstoppable-wallet-android#7661] [Q&A]
Hi, I am an Android Developer (Kotlin) 👋
I want to learn Web3 and Specially in terms of making Crypto Wallets. 
Have no deep idea about web3/blockchain(Tried to learn some but filled with question). Is there any possibility I can learn it fast and contribute to this project? 
if yes, please help me with some resources.

**Comment by worldwiseprofit-art** [comment]
Yes ,  please

**Comment by worldwiseprofit-art** [comment]
I need to watch a titular video to guide me. Or a good steps guide.
